libclc/generic/include/clc/clc.h

```diff
@@ -18,8 +18,16 @@
     (__OPENCL_C_VERSION__ >= CL_VERSION_3_0 &&                                 \
      defined(__opencl_c_generic_address_space))
 #define _CLC_GENERIC_AS_SUPPORTED 1
```

Comment:
Nit: I quickly grepped through the code, but is this macro still used after your change?
It looks like most of its uses have been replaced with `_CLC_DISTINCT_GENERIC_AS_SUPPORTED`.
I might be wrong here.

Reply:
Yep, that's right - all current uses of `_CLC_GENERIC_AS_SUPPORTED` have been replaced with `_CLC_DISTINCT_GENERIC_AS_SUPPORTED`. I'm keeping it as https://github.com/intel/llvm/pull/13428 will be making use of `_CLC_GENERIC_AS_SUPPORTED`